catalogue/aarch64-ifetch/tests/@all

```diff
@@ -0,0 +1,13 @@
+DIC0-IDC0/@all
+DIC0-IDC1/@all
+DIC1-IDC1/@all
+@jit
```

Comment:
File `@jit` has not been commited.

Reply:
Oops, my apologies -- and thanks for spotting this. Should be there now.